test/sql/include/plan_expand_hypertable_query.sql

```diff
@@ -141,19 +141,51 @@ SELECT * FROM cte ORDER BY value;
 
 \qecho test constraint exclusion for constraints in ON clause of JOINs
 
+-- Turn on chunk exclusion so that we can compare with how
+-- PostgreSQL does exclusion
+SET constraint_exclusion=on;
+SHOW constraint_exclusion;
+
 \qecho should exclude chunks on m1 and propagate qual to m2 because of INNER JOIN
+SET timescaledb.enable_constraint_exclusion = off;
+:PREFIX SELECT m1.time,m2.time FROM metrics_timestamptz m1 INNER JOIN metrics_timestamptz m2 ON m1.time = m2.time AND m1.time < '2000-01-10' ORDER BY m1.time;
+SET timescaledb.enable_constraint_exclusion = on;
 :PREFIX SELECT m1.time,m2.time FROM metrics_timestamptz m1 INNER JOIN metrics_timestamptz m2 ON m1.time = m2.time AND m1.time < '2000-01-10' ORDER BY m1.time;
+
 \qecho should exclude chunks on m2 and propagate qual to m1 because of INNER JOIN
+SET timescaledb.enable_constraint_exclusion = off;
+:PREFIX SELECT m1.time,m2.time FROM metrics_timestamptz m1 INNER JOIN metrics_timestamptz m2 ON m1.time = m2.time AND m2.time < '2000-01-10' ORDER BY m1.time;
+SET timescaledb.enable_constraint_exclusion = on;
 :PREFIX SELECT m1.time,m2.time FROM metrics_timestamptz m1 INNER JOIN metrics_timestamptz m2 ON m1.time = m2.time AND m2.time < '2000-01-10' ORDER BY m1.time;
+
 \qecho must not exclude on m1
+SET timescaledb.enable_constraint_exclusion = off;
+:PREFIX SELECT m1.time,m2.time FROM metrics_timestamptz m1 LEFT JOIN metrics_timestamptz m2 ON m1.time = m2.time AND m1.time < '2000-01-10' ORDER BY m1.time;
+SET timescaledb.enable_constraint_exclusion = on;
 :PREFIX SELECT m1.time,m2.time FROM metrics_timestamptz m1 LEFT JOIN metrics_timestamptz m2 ON m1.time = m2.time AND m1.time < '2000-01-10' ORDER BY m1.time;
```

Comment:
CREATE TABLE metrics_timestamp_2 as select * from metrics_timestamp;
SELECT create_hypertable(...);

Then use this new table for LEFT and RIGHT JOIN test cases

```suggestion
:PREFIX SELECT m1.time,m2.time FROM metrics_timestamptz m1 LEFT JOIN metrics_timestamptz_2 m2 ON m1.time = m2.time AND m1.time < '2000-01-10' ORDER BY m1.time;
```